pkg/awsconfig/testing/firewallConfig.go

```diff
@@ -0,0 +1,479 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+	"github.com/aws/aws-sdk-go/service/networkfirewall"
+	"github.com/aws/aws-sdk-go/service/networkfirewall/networkfirewalliface"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+type Client struct {
+	ec2Client      ec2iface.EC2API
+	firewallClient networkfirewalliface.NetworkFirewallAPI
+}
+
+var wait2 = 360 * time.Second
+
+
+func main() {
+	var ec2Client ec2iface.EC2API
+	var firewallClient networkfirewalliface.NetworkFirewallAPI
+
+	region := os.Getenv("REGION")
+	profile := os.Getenv("PROFILE")
```

Comment:
Thought about supporting command line args here?

Reply:
well since the amount of argument is different ( passing PROFILE vs passing ENV) i think it might be safer like this than command lines args

Reply:
```
AWS_ACCESS_KEY_ID=foo AWS_SECRET_ACCESS_KEY=bar AWS_SESSION_TOKEN=baz  ./firewall-create

AWS_PROFILE=devaccount AWS_REGION=us-west-2 ./firewall-create

firewall-create --profile devaccount --region us-east-1
```

All three of these should work

Reply:
implemented -p $profile -r $region